packages/react-scripts/config/webpackDevServer.config.js

```diff
@@ -8,14 +8,14 @@
 // @remove-on-eject-end
 'use strict';
 
+const fs = require('fs');
```

Comment:
Where is this used? I guess it can be removed? :)

Reply:
I actually just moved this from line 16 so the relative imports were at the end. It's used by other things in this file.

Reply:
Thanks!